Pages/Response.razor

```diff
@@ -0,0 +1,17 @@
+@page "/response"
+
+<h1>Response</h1>
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ChatGPT Response</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
```

Comment:
All of this can be removed. See [this review comment](https://github.com/QuinnMatthews/EthicsQA-App/pull/4#discussion_r1566213355)